include/crm/crm.h

```diff
@@ -187,6 +187,7 @@ extern char *crm_system_name;
 #  define RSC_NOTIFIED	CRMD_ACTION_NOTIFIED
 
 #  define RSC_STATUS	CRMD_ACTION_STATUS
+#  define RSC_PROBE	CRMD_ACTION_STATUS"(probe)"
```

Comment:
Thank you for comment !
I did not know a lot about processing of preprocesser.
I intend to change it as follows.
```
#  define CRMD_ACTION_PROBE            "monitor(probe)"
```